### PR TITLE
fix adding new bots

### PR DESCRIPTION
**Pull request by ghost**
fixes certtools/intelmq-manager#279

### CODE REVIEW

**Review by monoidic — COMMENTED** [review]
If it's a new bot, then we should still populate `node.parameters.destination_queues` to an empty object. Hence, I'd suggest:
```js
node.parameters.destination_queues = old_bot ? old_bot.parameters.destination_queues : {};
```

File: intelmq_manager/static/js/configs.js [hunk 1]
```diff
@@ -558,7 +558,9 @@ function saveData(data, callback) {
     let current_id = node.bot_id, old_id = app.bot_before_altering.bot_id;
 
     let old_bot = app.nodes[old_id];
-    node.parameters.destination_queues = old_bot.parameters.destination_queues;
+    if(old_bot != undefined) {
+        node.parameters.destination_queues = old_bot.parameters.destination_queues;
+    }
```
Comment (monoidic) [inline intelmq_manager/static/js/configs.js]:
```suggestion
    node.parameters.destination_queues = old_bot ? old_bot.parameters.destination_queues : {};
```

**Review by monoidic — APPROVED** [review]
Fixes it in my env.

**Comment by ghost** [comment]
Thx!

**Pull request merged**